### PR TITLE
Write away from the flakey array_filter and check more deliberately

### PR DESCRIPTION
**Pull request by borkweb**
See: https://central.tri.be/issues/61940#note-18

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Don't leave debug_robot lying around | [Migrate.php:64](https://github.com/moderntribe/the-events-calendar/blob/c4ed8413eb6a1bcb3594ef4c4b858a2ae62380cd/src/Tribe/Aggregator/Migrate.php#L64)
:x: | Don't leave debug_robot lying around | [Migrate.php:65](https://github.com/moderntribe/the-events-calendar/blob/c4ed8413eb6a1bcb3594ef4c4b858a2ae62380cd/src/Tribe/Aggregator/Migrate.php#L65)

> via jenkins/codesniffer-phpcs

**Comment by bordoni** [comment]
:10

File: src/Tribe/Aggregator/Migrate.php [hunk 1]
```diff
@@ -245,6 +245,30 @@ public function is_ical_migrated() {
...
+	 * Filters out empty values
+	 *
+	 * NOTE: we aren't using array_filter because EVEN with an empty() alias, the results are
+	 * unpredictable
+	 *
+	 * @param array $original_values
+	 *
+	 * @return array
+	 */
+	public function filter_out_unwanted_values( $original_values ) {
```
Comment (barryhughes) [inline src/Tribe/Aggregator/Migrate.php]: Purely for my own ref I'd love to know more about when and where array_filter was letting us down here. That aside, as a general purpose helper that could be useful in all kinds of places, might it be better housed someplace inside tribe-common?

Comment (borkweb) [inline src/Tribe/Aggregator/Migrate.php]: Yeah, something along this vein may have a nice life in tribe-common - but I don't think this iteration as-is is as flexible as we could make it. In the interest of moving to Smoketest, I'm going to leave this here and puzzle over a better tribe-common method at a later date.

**Pull request merged**